modules/virt-configuring-certificate-rotation.adoc

```diff
@@ -0,0 +1,42 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-configuring-certificate-rotation.adoc
+
+[id="virt-configuring-certificate-rotation_{context}"]
+
+Configure certificate rotation parameters to replace existing certificates. You can do this during OpenShift Virtualization installation in the web console or after installation in the YAML file.
```

Comment:
s/OpenShift Virtualization/{VirtProductName}

s/in the YAML file./in the `HyperConverged` custom resource (CR).